core/src/main/java/org/apache/iceberg/BaseRewriteFiles.java

```diff
@@ -23,8 +23,13 @@
 import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
 
 class BaseRewriteFiles extends MergingSnapshotProducer<RewriteFiles> implements RewriteFiles {
+
```

Comment:
Nit: we try to avoid whitespace changes because they can easily cause unnecessary commit conflicts.